### PR TITLE
Rebalances Yautja bracer power

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
The bracer power supply has been rebalanced, reducing the cost of creating crystals and health capsules due to scarcity of recharging, and the fact thwei is no longer as powerful. The plasma caster has also had its power costs reduced slightly.

# Explain why it's good for the game
Allows Yautja to operate a bit longer before they need to worry about recharging.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
balance: Reduces power cost for plasma caster use, and creation of thwei crystals/health capsules.
/:cl:

### CODE REVIEW

File: code/modules/cm_preds/yaut_weapons.dm [hunk 1]
```diff
@@ -1164,15 +1168,15 @@
 			switch(strength)
 				if("low power stun bolts")
 					strength = "high power stun bolts"
-					charge_cost = 100
-					set_fire_delay(FIRE_DELAY_TIER_6 * 3)
+					charge_cost = 50
+					set_fire_delay(FIRE_DELAY_TIER_1)
```
Comment (Zonespace27) [inline code/modules/cm_preds/yaut_weapons.dm:1172]: This is a pretty big change delay-wise, double checking that it's intentional w/ this PR

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by morrowwolf** [comment]
@realforest2001 this has a review question btw

Otherwise I don't care and it's good to merge.

File: code/modules/cm_preds/yaut_weapons.dm (hunk 1 above)
Comment (realforest2001) [reply code/modules/cm_preds/yaut_weapons.dm:1172]: intended

**Pull request merged**